grails-gsp/src/main/groovy/org/grails/gsp/GroovyPagesTemplateEngine.java

```diff
@@ -201,14 +201,6 @@ public void setJspTagLibraryResolver(TagLibraryResolver jspTagLibraryResolver) {
         this.jspTagLibraryResolver = jspTagLibraryResolver;
     }
 
-    /**
```

Comment:
Is there any specific reason to remove the setter?

Reply:
The discussion started [here](https://github.com/spring-projects/spring-framework/issues/28261#issuecomment-1085583295) in spring-framework should shed some light on this.